src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs

```diff
@@ -244,13 +247,26 @@ internal IEnumerable<FunctionMetadata> ValidateMetadata(IEnumerable<RawFunctionM
             return validatedMetadata;
         }
 
+        private static BindingMetadata GetBindingMetadata(string binding)
+        {
+            using (StringReader stringReader = new StringReader(binding))
+            using (JsonTextReader jsonTextReader = new JsonTextReader(stringReader))
+            {
+                JsonSerializer serializer = JsonSerializer.Create(_jsonSerializerSettings);
```

Comment:
Can we DI the JsonSerializer so we're not creating one every time this method is called?